src/tools/mail.py

```diff
@@ -0,0 +1,72 @@
+import smtplib
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from email.mime.application import MIMEApplication
+from email.utils import formatdate
+import os
+
+class Email:
+    def __init__(self):
+        """Initialize the Email class."""
+        self.password = None
+
+    def set_password(self, password):
+        """Set the email account password."""
+        self.password = password  # Removed unnecessary 'pass'
+
+    def send_email(self, subject, body, to_email, from_email, smtp_server='smtp.gmail.com', smtp_port=587):
```

Comment:
don't fix the stmp_server 
```suggestion
    def send_email(self, subject, body, to_email, from_email, smtp_port=587):
    smtp_server = self.smtp_server
```